CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Developing XTF
+
+Compile from source:
+
+```mvn clean install```
+
+# XTF git development cycle <a name="devcycle"></a>
+We use the [git flow](https://guides.github.com/introduction/flow/) workflow.
+The steps are
+
+0. Read the [guide](https://guides.github.com/).
+1. Register on [GitHub](https://github.com/).
+2. Fork the XTF repository.
+3. Clone your fork, add the original XTF develop repository as a remote, and check out locally
```

Comment:
What does XTF develop mean? To me, it would make sense if you simply said "original XTF repository".

Reply:
I've used `upstream XTF repository` as original might be confusing as it's default name for user's fork.